routes/authRoutes.js

```diff
@@ -36,7 +36,7 @@ router.post('/login', async (req, res) => {
 
 	try {
 		// Searching for a single user in the database, with the email provided in the request body. 
-		const user = await UserModel.findOne({ email: req.body.email.toLowerCase() });		
+		const user = await UserModel.findOne({ email: new RegExp(`^${req.body.email}$`, 'i')  });		
```

Comment:
Any reason for replacing the .toLowerCase() here?

Reply:
Yes! `.findOne()` is case sensitive so accounts with an email that contained upper case characters would never be found and therefore wouldnt be able to log in. The bug was introduced here https://github.com/Educado-App/educado-backend/commit/950b3f58d29b0f16bd4cc54bc8342b4c2c8d986e#diff-f85f695e725f0f6ed629f83a47b27c9eff98786402a1ab5373614253dc93199bR39

@Todua992 What was the bug you fixed and how do we make login work again without regressing (in relation to whatever you fixed)?

Reply:
Perhaps a better solution is to lowercase mails when registrering new users, since mails normally are case insensitive anyways this way, we can maintain the current findOne for users?

Reply:
Whoops i can see that's exactly what you did already, just got send directly to this file change on mobile through mail mb.

Reply:
Looks like all the users on the remote database have their emails lower cased and all accounts work so it looks like this doesnt apply 🤔 It broke my local development environment though so i thought it was broken everywhere and there wasnt any notification for people to update their environments.
I spent a good while debugging this, so im a little annoyed.